torch/fx/experimental/proxy_tensor.py

```diff
@@ -403,7 +421,7 @@ def wrap_fake(x):
         if use_fake:  # type: ignore[attr-defined]
             args = pytree.tree_map(wrap_fake, args)
 
-        with decompose(decomposition_table), fake_tensor_mode, proxy_mode:  # type: ignore[attr-defined]
+        with decompose(decomposition_table), fake_tensor_mode, proxy_mode, proxy_dispatch_mode_ctx(proxy_mode):  # type: ignore[attr-defined] # noqa: B950
```

Comment:
is there a better way to handle this?

- if I use a backslash for line continuation, there's no way to apply `# type: ignore[attr-defined]` to suppress mypy error
- `with (a, b, c, d):` allows splitting the line to <140 char/line and `#type: ignore[attr-defined]` works with it, but it only works in python 3.9+
- `contextlib.nested` is deprecated

Reply:
you could use ExitStack if you like